tests/main/views/test_buyers.py

```diff
@@ -85,15 +80,20 @@ def find_briefs_mock():
     return find_briefs_response
 
 
-@mock.patch('app.main.views.buyers.data_api_client', autospec=True)
 class TestBuyerDashboard(BaseApplicationTest):
 
     def setup_method(self, method):
-        super(TestBuyerDashboard, self).setup_method(method)
+        super().setup_method(method)
+        self.data_api_client_patch = mock.patch('app.main.views.buyers.data_api_client', autospec=True)
+        self.data_api_client = self.data_api_client_patch.start()
+        self.data_api_client.find_briefs.return_value = find_briefs_mock()
```

Comment:
This is a nice way of doing a patch. It's a little less obvious but it's cleaner.